### DISCUSSION TITLE
Admin Claims depth

### DISCUSSION THREAD

**Discussion by Neraum** [GriefPrevention/GriefPrevention#1926] [Q&A]
Love the plugin but the depth of claims is a bit confusing. It only goes deeper when the owner of the claim builds downwards, nobody who is trusted can push the depth lower. So how do Admin claims go deeper, or are they permanently set to the depth in the config? Which I don't even mind, means I can build stuff on the nether roof but people can still dig below it

**Comment by Jikoo** [comment]
Anyone who is allowed to build in the claim will cause it to extend, not just the owner. If you or anyone else with admin perms builds beneath an admin claim, it will extend.
https://github.com/TechFortress/GriefPrevention/blob/058ae98ceea307cf72155bde421564080ac85eb1/src/main/java/me/ryanhamshire/GriefPrevention/BlockEventHandler.java#L313-L318
